conn/physic/units.go

```diff
@@ -352,6 +354,111 @@ const (
 	Fahrenheit      Temperature = 555555555 * NanoKelvin
 )
 
+// Power is a measurement of power stored as a nano watts.
+//
+// The highest representable value is 9.2GW.
+type Power int64
+
+// String returns the power formatted as a string in watts.
+func (p Power) String() string {
+	return nanoAsString(int64(p)) + "W"
+}
+
+const (
+	//Watt is unit of power J/s, kg⋅m²⋅s⁻³
+	NanoWatt  Power = 1
+	MicroWatt Power = 1000 * NanoWatt
+	MilliWatt Power = 1000 * MicroWatt
+	Watt      Power = 1000 * MilliWatt
+	KiloWatt  Power = 1000 * Watt
+	MegaWatt  Power = 1000 * KiloWatt
+	GigaWatt  Power = 1000 * MegaWatt
+)
+
+// Energy is a measurement of work stored as a nano joules.
+//
+// The highest representable value is 9.2GJ.
+type Energy int64
+
+// String returns the energy formatted as a string in Joules.
+func (e Energy) String() string {
+	return nanoAsString(int64(e)) + "J"
+}
+
+const (
+	//Joule is a unit of work. kg⋅m2⋅s⁻²
+	NanoJoule  Energy = 1
+	MicroJoule Energy = 1000 * NanoJoule
+	MilliJoule Energy = 1000 * MicroJoule
+	Joule      Energy = 1000 * MilliJoule
+	KiloJoule  Energy = 1000 * Joule
+	MegaJoule  Energy = 1000 * KiloJoule
+	GigaJoule  Energy = 1000 * MegaJoule
+)
+
+// Capacitance is a measurement of capacitance stored as a pico farad.
+//
+// The highest representable value is 9.2MF.
+type Capacitance int64
+
+// String returns the energy formatted as a string in Joules.
+func (c Capacitance) String() string {
+	return picoAsString(int64(c)) + "F"
+}
+
+const (
+	//Farad is a unit of capacitance. kg⁻¹⋅m⁻²⋅s⁴A²
+	PicoFarad  Capacitance = 1
+	NanoFarad  Capacitance = 1000 * PicoFarad
+	MicroFarad Capacitance = 1000 * NanoFarad
+	MilliFarad Capacitance = 1000 * MicroFarad
+	Farad      Capacitance = 1000 * MilliFarad
+	KiloFarad  Capacitance = 1000 * Farad
+	MegaFarad  Capacitance = 1000 * KiloFarad
+)
+
+// LuminousIntensity is a measurement of wavelength power stored as a nano candela.
+//
```

Comment:
Worth adding: `This is one of the base unit in the International System of Units.` as done for the other base ones.